Sources/DashX/DashXClient.swift

```diff
@@ -63,23 +63,23 @@ public class DashXClient {
         }
     }
 
-    func setPublicKey(to publicKey: String) {
+    public func setPublicKey(to publicKey: String) {
         ConfigInterceptor.shared.publicKey = publicKey
     }
     
-    func setBaseURI(to baseURI: String) {
+    public func setBaseURI(to baseURI: String) {
         Network.shared.setBaseURI(to: baseURI)
     }
     
-    func setTargetEnvironment(to environment: String) {
+    public func setTargetEnvironment(to environment: String) {
         ConfigInterceptor.shared.targetEnvironment = environment
     }
-
-    func setTargetInstallation(to targetInstallation: String) {
-        ConfigInterceptor.shared.targetInstallation = targetInstallation
+    
+    public func setAccountUid(with uid: String) {
+        self.uid = uid
```

Comment:
Rename the private variables: https://github.com/dashxhq/dashx-browser/blob/main/src/Client.ts#L28-L32